cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -84,11 +84,11 @@ func NewKubeStaticPodPathManagerUsingTempDirs(realManifestDir string, saveManife
 	if err != nil {
 		return nil, err
 	}
-	backupManifestsDir, err := constants.CreateTimestampDirForKubeadm("kubeadm-backup-manifests")
```

Comment:
why did this function change?

Reply:
- This is probably a mistake I made while resolving conflicts on a rebase
- The fact that all test came back green suggests that this is still ok
- I assume, that there is some reason why this change was made and something/someone is expecting a timestamped directory - if so, I think there should be some sort of test around that then
- I will try to add a test for that (if meaningful and reasonably possible -- nee to check) to guard against that

Reply:
I'm ok with omitting a test, that is out of scope for this PR to me, if you remove this change then it all looks good to me!